apps/aecore/test/aecore_txs_SUITE.erl

```diff
@@ -150,7 +150,7 @@ txs_gc(Config) ->
     aecore_suite_utils:mine_key_blocks(N1, 2),
 
     %% Now there should be no transactions in mempool
-    {ok, Txs6} = rpc:call(N1, aec_tx_pool, peek, [infinity]),
+    {ok, Txs6} = pool_peek(N1),
```

Comment:
I understand there are:
* The RPC call aec_conductor:stop_mining in mine_blocks that casts garbage_collect to aec_tx_pool that spawns (let us call this process X) that sends new_gc_height back to aec_tx_pool that casts garbage_collect to aec_tx_pool_gc.
* The test calling peek here.

This change adds a synchronous call to aec_tx_pool_gc, before peeking aec_tx_pool.

It looks to me that test could still fail if e.g.:
* process X is slow; or
* aec_tx_pool_gc is slow in performing the actual GC.

So it looks like it would be more robust and readable to add a wait-until with a short time and quick polling here?

Though an improvement so ok.